ocaml/alerts/expiry_alert.mli

```diff
@@ -0,0 +1,85 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+type message_name_t = string
+
+type message_priority_t = int64
+
+(* message_id_t is the type of mesage defined in Api_messages *)
+type message_id_t = message_name_t * message_priority_t
+
+type remaining_days_t = int
+
+type raw_alert_t = {
+    cls:
```

Comment:
This is a polymorphic variant and hence extensible. Is this flexibility required and could this be a regular variant instead?

Reply:
I think this is aligned with https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/xapi_message.ml#L413, which is defined in https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi-cli-server/record_util.ml#L28